dist/lozad.es.js

```diff
@@ -94,7 +94,7 @@ const getElements = (selector, root = document) => {
 };
 
 function lozad (selector = '.lozad', options = {}) {
-  const {root, rootMargin, threshold, load, loaded} = {...defaultConfig, ...options};
```

Comment:
any specific reason to remove this up?

Reply:
I didn't remove that, not manually/deliberately at least.
After changing the relevant part (line 48 to 50 in lozad.js/src/lozad.js), 
I've issued following commands:
```bash
npm run lint
npm run test
npm run build
```
Then pushed the PR up simply by
```bash
git push --set-upstream origin multiple-background-images
```

Nothing else, I'm positive.
I'm not sure about what went wrong.

It's hard to deal with a newbie, huh?